<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>740</width>
      <height>560</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_rawimage">
     <attribute name="title">
      <string>RawImage</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>195</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_height">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_height"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_dgain">
         <property name="text">
          <string>Dgain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dgain"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>143</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_pixeloffset">
         <property name="text">
          <string>PixelOffset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_pixeloffset"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>143</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_inputformat">
         <property name="text">
          <string>InputFormat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_inputformat"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>149</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_outputformat">
         <property name="text">
          <string>OutputFormat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_outputformat"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>137</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_dataformat">
         <property name="text">
          <string>DataFormat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_dataformat"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_cfa">
         <property name="text">
          <string>CFA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_cfa"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_rawtype">
         <property name="text">
          <string>RawType</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_rawtype"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_viewrawfile">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ViewRawFile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_raw2plain16">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Raw2Plain16</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_raw2dng">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Raw2Dng</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_bitshift">
         <property name="text">
          <string>BitShift</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_bitshift"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>621</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_dir">
         <property name="text">
          <string>Dir:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_dir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addfile">
         <property name="text">
          <string>AddFile...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>195</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_width">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_width"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cmos_sensor">
     <attribute name="title">
      <string>CmosSensor</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_snr_linear">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SNR/Linear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_dynamicrange">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Dynamic Range</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_temporalnoise">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Temporal Noise</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_isp_pipeline">
     <attribute name="title">
      <string>IspPipeLine</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_isppipeline">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>isppipeline</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_image_quality">
     <attribute name="title">
      <string>ImageQuality</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_imagequality">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>imagequality</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
